launch/launch/logging/__init__.py

```diff
@@ -358,7 +365,8 @@ def get_log_file_handler(file_name='launch.log'):
             else:
                 launch_config.log_handler_factory = handlers.FileHandler
         file_path = get_log_file_path(file_name)
-        file_handler = launch_config.log_handler_factory(file_path)
+        file_handler = launch_config.log_handler_factory(
+            file_path, encoding='utf-8')
```

Comment:
@dirk-thomas why do we force `UTF-8` for files and use `locale.getpreferredencoding()` for standard streams only?

Reply:
We usually write all files in UTF-8. The standard streams have their own encoding which we have to use as is.

Reply:
Fair enough.